pkg/vrct/vrctFs/vrct.go

```diff
@@ -1,28 +1,59 @@
 package vrctFs
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
+	"github.com/pelletier/go-toml/v2"
 	"gopkg.in/mgo.v2/bson"
+	"gopkg.in/yaml.v3"
 	"os"
-	"slices"
+	"path/filepath"
+	"reflect"
 	"sort"
 	"strings"
 )
 
-const VIRTUAL_FS_PATH_PREFIX = "/tmp/spito-vrct/fs"
+const VirtualFsPathPrefix = "/tmp/spito-vrct/fs"
+const VirtualFilePostfix = ".prototype.bson"
 
 type FsVRCT struct {
 	virtualFSPath  string
 	fsRequirements []FsRequirement
 }
 
+type FsRequirement struct {
+	// In simpler words - how this rule appeared here
+	ruleStackTrace   []string
```

Comment:
I see this isn't implemented. You should add TODO comment

Reply:
Let's remove it